tests/ui/test_utils.py

```diff
@@ -158,7 +158,7 @@ def test_is_muted(mocker, msg, narrow, muted_streams, muted_topics, muted):
         [1],
         None,
         True,
-        0,
+        1,
```

Comment:
To confirm: this is since in a muted stream, we're now showing the message?

We should probably add a test for this case; now if you feel it's straightforward, or in a follow-up PR perhaps.

Reply:
Yes. That's the reason it was changed to `1`.
Sure. Will add a test for this case.